internal/versions/versions.go

```diff
@@ -382,8 +382,13 @@ func (h *handler) getSupportLevel(openshiftVersion string) string {
 
 // Ensure no missing values in OS images and Release images.
 func (h *handler) validateVersions() error {
+	openshiftVersions := h.GetOpenshiftVersions()
+	if len(openshiftVersions) == 0 {
+		return errors.Errorf("No OS images are available")
+	}
```

Comment:
Do we want to allow deploying the operator without any versions and add them in runtime?